inspirehep/config.py

```diff
@@ -85,6 +85,12 @@ def _(x):
 CACHE_TYPE = "redis"
 ACCOUNTS_SESSION_REDIS_URL = "redis://localhost:6379/2"
 
+# Mathoid
+# =======
+MATHOID_SERVER = os.environ.get(
```

Comment:
:+1: That very precise change was sitting in my laptop, not yet pushed to :-)

In particular since I am not going to fiddle with npm and honcho I know many people will not have mathoid running...

Additionally I think I will separate mathoid testing from the integration ones.